drift/lib/src/runtime/query_builder/schema/entities.dart

```diff
@@ -47,28 +47,6 @@ class Index extends DatabaseSchemaEntity {
   Index(this.entityName, this.createIndexStmt);
 }
 
-/// A sqlite view.
-///
-/// In drift, views can only be declared in `.drift` files.
-///
-/// For more information on views, see the [CREATE VIEW][sqlite-docs]
-/// documentation from sqlite, or the [entry on sqlitetutorial.net][sql-tut].
-///
-/// [sqlite-docs]: https://www.sqlite.org/lang_createview.html
-/// [sql-tut]: https://www.sqlitetutorial.net/sqlite-create-view/
-abstract class View<Self, Row> extends ResultSetImplementation<Self, Row>
```

Comment:
This is necessary. It should have been called `ViewInfo` originally as tables do. Because of caret versioning may generated code breaks on current users without regenerate. Can we rename current `View` to (for ex.) `DslView`, but I think the current `View` name would be better to understand and users will regenerate the code if see compiler fails on there. Not a big deal.